smartsim/_core/config/config.py

```diff
@@ -155,20 +155,27 @@ def database_file_parse_interval(self) -> int:
 
     @property
     def dragon_server_path(self) -> t.Optional[str]:
-        return os.getenv("SMARTSIM_DRAGON_SERVER_PATH")
-
-    @property
-    def dragon_server_reconnect_timeout(self) -> int:
-        return int(os.getenv("SMARTSIM_DRAGON_RECONNECT_TIMEOUT", "5000"))
+        return os.getenv(
+            "SMARTSIM_DRAGON_SERVER_PATH",
+            os.getenv("SMARTSIM_DRAGON_SERVER_PATH_EXP", None),
+        )
 
     @property
     def dragon_server_timeout(self) -> int:
-        return int(os.getenv("SMARTSIM_DRAGON_TIMEOUT", "5000"))
+        return int(os.getenv("SMARTSIM_DRAGON_TIMEOUT", "20000"))
 
     @property
     def dragon_server_startup_timeout(self) -> int:
         return int(os.getenv("SMARTSIM_DRAGON_STARTUP_TIMEOUT", "-1"))
 
+    @property
+    def dragon_transport(self) -> str:
+        return os.getenv("SMARTSIM_DRAGON_TRANSPORT", "hsta")
+
+    @property
+    def dragon_log_level(self) -> str:
+        return os.getenv("SMARTSIM_DRAGON_LOG_LEVEL", "NONE")
```

Comment:
I'm on the fence about adding a new log level env var for dragon. What's blocking you from just using `SMARTSIM_LOG_LEVEL`? It might be really nice to only have to change/remember one value

Reply:
It's just that `DEBUG` level in Dragon creates a lot of files (up to 40 MB I think) and I'd prefer being a bit more conservative

Reply:
Maybe we need some mapping, like `developer`->`DEBUG`, while `debug`-><something less verbose>

Reply:
Did in the `DragonConnector` class